Packs/Okta/Integrations/Okta_IAM/Okta_IAM.py

```diff
@@ -0,0 +1,399 @@
+                iden=okta_user.get('id'),
+                email=okta_user.get('profile', {}).get('email'),
+                username=okta_user.get('profile', {}).get('login'),
+                details=okta_user
+            )
+
+    except DemistoException as e:
+        handle_exception(user_profile, e, IAMActions.GET_USER)
+
+    return user_profile
+
+
+def enable_user_command(client, args, mapper_out, is_command_enabled, is_create_user_enabled):
+    if not is_command_enabled:
+        return None
+
+    user_profile = IAMUserProfile(user_profile=args.get('user-profile'))
+    try:
+        okta_user = client.get_user(user_profile.get_attribute('email'))
+        if not okta_user:
+            if args.get('create-if-not-exists').lower() == 'true':
+                user_profile = create_user_command(client, args, mapper_out, is_create_user_enabled)
+            else:
+                _, error_message = IAMErrors.USER_DOES_NOT_EXIST
+                user_profile.set_result(action=IAMActions.ENABLE_USER,
+                                        skip=True,
+                                        skip_reason=error_message)
+        else:
+            client.activate_user(okta_user.get('id'))
+            user_profile.set_result(
+                action=IAMActions.ENABLE_USER,
+                success=True,
+                active=True,
+                iden=okta_user.get('id'),
+                email=okta_user.get('profile', {}).get('email'),
+                username=okta_user.get('profile', {}).get('login'),
+                details=okta_user
+            )
+
+    except DemistoException as e:
+        handle_exception(user_profile, e, IAMActions.ENABLE_USER)
+
+    return user_profile
+
+
+def disable_user_command(client, args, is_command_enabled):
+    if not is_command_enabled:
+        return None
+
+    user_profile = IAMUserProfile(user_profile=args.get('user-profile'))
+    try:
+        okta_user = client.get_user(user_profile.get_attribute('email'))
+        if not okta_user:
+            _, error_message = IAMErrors.USER_DOES_NOT_EXIST
+            user_profile.set_result(action=IAMActions.DISABLE_USER,
+                                    skip=True,
+                                    skip_reason=error_message)
+        else:
+            client.deactivate_user(okta_user.get('id'))
+            user_profile.set_result(
+                action=IAMActions.DISABLE_USER,
+                success=True,
+                active=False,
+                iden=okta_user.get('id'),
+                email=okta_user.get('profile', {}).get('email'),
+                username=okta_user.get('profile', {}).get('login'),
+                details=okta_user
+            )
+
+    except DemistoException as e:
+        handle_exception(user_profile, e, IAMActions.DISABLE_USER)
+
+    return user_profile
+
+
+def create_user_command(client, args, mapper_out, is_command_enabled):
+    if not is_command_enabled:
+        return None
```

Comment:
Return string indicating create user disabled in Okta IAM, skippping creating user for {email}